### PR TITLE
update firefox-cliqz notice with 56 formatting

### PR DESCRIPTION
**Pull request by eleemoz**

### CODE REVIEW

**Comment by peiying2** [comment]
@eleemoz I assume this needs to be updated in German too.  I know since Casey left, can we make sure I am alerted with any legal updates that impact l10n?  Should the German one be updated ASAP?  If there is any simultaneous release, heads-up and sufficient time for l10n to process is very important.

**Comment by schalkneethling** [comment]
Oh so, I see it is merged already. Seeing that the Cliqz privacy policy page still uses the old format, if this goes to production the page will not display as intended. Is there currently a bug to have the Firefox Cliqz privacy page much the Firefox one?

File: firefox-cliqz_privacy_notice/en-US.md [hunk 1]
```diff
@@ -1,137 +1,176 @@
...
+{: #referraltracking }
 
-	Search suggestions is a feature to help you to find out common phrases that other people have searched for. These search suggestions are offered by your Default Search Engines (such as Google, Yahoo, etc.) and not by Firefox. If you enable this feature, and your Default Search Engine supports suggestions, Firefox may send the terms you type in the Awesome Bar or Search Bar to your Default Search Engine to retrieve suggestions, and is governed by the applicable Privacy Policy from your Default Search Engine. You can [learn more about Search Suggestions here](https://support.mozilla.org/kb/use-popular-search-suggestions-firefox-search-bar) and how to enable or disable it.
+    _On Desktop_: Firefox by default sends Mozilla HTTP data that may be included with Firefox’s installer.  This enables us to determine the website domain or advertising campaign (if any) that referred you to our download page. Read the [documentation](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/environment.html#attribution) or [opt-out](https://support.mozilla.org/kb/desktop-privacy) before installation.
```
Comment (schalkneethling) [inline firefox-cliqz_privacy_notice/en-US.md]: This should not be indented.

File: firefox-cliqz_privacy_notice/en-US.md [hunk 2]
```diff
@@ -1,137 +1,176 @@
...
 
-* **Referral and Campaign Tracking**{: #thirdparty } ** **{: #referraltracking }
+    _On iOS and Android_: Firefox by default sends mobile campaign data to Adjust, our analytics vendor, which has its own [privacy policy](https://www.adjust.com/privacy_policy/).  Mobile campaign data includes a Google advertising ID, IP address, timestamp, country, language/locale, operating system, and app version. Read the [documentation](https://firefox-source-docs.mozilla.org/mobile/android/fennec/adjust.html).
```
Comment (schalkneethling) [inline firefox-cliqz_privacy_notice/en-US.md]: This should not be indented.

File: firefox-cliqz_privacy_notice/en-US.md [hunk 3]
```diff
@@ -1,137 +1,176 @@
...
-* **Firefox Health Report**
-{: #health-report .inproduct-link }
+* __Interaction Data to Mozilla__: This includes counts of visits to search engine pages, which search engines you use, and your Cliqz identifier.  
+	
+[Learn more](https://cliqz.com/en/whycliqz/transparency) about Cliqz data collection. To check if the Cliqz feature is enabled or to disable data collection:
+  * type "about:addons" into your Firefox URL bar
+  * select "Extensions" on the left menu bar
+  * if installed, Cliqz will appear on the right
+  * selecting "Disable" will turn off the Cliqz feature and stop related data collection
+  * selecting "Remove" will uninstall the Cliqz feature from Firefox
```
Comment (schalkneethling) [inline firefox-cliqz_privacy_notice/en-US.md]: I will need to test this locally to be 100% sure but, my first instinct is that you probably want an empty line before the first bullet, and do not indent them.
